spec/hipchat_api_v2_spec.rb

```diff
@@ -379,6 +379,15 @@
     end
   end
 
+  describe "#delete_room" do
```

Comment:
Sure. I've done it for the delete test. There's no such test for any other endpoint and I haven't added them though.